packages/core/src/languages/solidity/predeploys.ts

```diff
@@ -175,6 +220,21 @@ export const deploySphinxSystem = async (
   for (const relayer of relayers) {
     const chainId = (await provider.getNetwork()).chainId
 
+    const currentDripVersion = fetchDripVersionForNetwork(chainId)
+    const baseDripName = `sphinx_fund_${relayer}`
+    const dripName =
+      baseDripName + (currentDripVersion > 0 ? `_${currentDripVersion}` : '')
+
+    // Cancel any out of date drips
+    await cancelPreviousDripVersions(
+      Drippie,
+      provider,
+      owner,
+      executionMode,
+      baseDripName,
+      currentDripVersion
+    )
```

Comment:
Handles cancelling the previous drip versions if necessary, the latest version is added below.